### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by ckoppenhaver**
This is the initial commit, the library is currently a work in progress that requires some more functionality to be ported over. Also the README needs fleshed out with instructions on how to use the library.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "phase2/svgo-php",
+  "description": "This is a PHP port of the SVGO [ https://github.com/svg/svgo ] library to optimize SVG vector graphic files.",
+  "require": {
```
Comment (grayside) [inline composer.json]: This is a bit light on typical details. See https://getcomposer.org/doc/04-schema.md for the formal spec. Look around at packages for typical stuff to include.

File: src/WWVSIconSVGOptimizer.class.inc [hunk 2]
```diff
@@ -0,0 +1,1605 @@
...
+ *    (See $this->setDefaultOptions().) However, the Sanitizer minify option
+ *    does not remove empty text elements, which we are doing in this class.
+ *    (See $this->removeEmptyText().)
+ *
+ */
+
+libraries_load('svg-sanitizer');
+
+module_load_include('inc', 'wwvs_icon_svg', 'includes/WWVSIconSVGOptimizerAttributes.class');
+module_load_include('inc', 'wwvs_icon_svg', 'includes/WWVSIconSVGOptimizerTags.class');
```
Comment (grayside) [inline src/WWVSIconSVGOptimizer.class.inc:71]: Drupal code?

File: src/WWVSIconSVGOptimizer.class.inc [hunk 3]
```diff
@@ -0,0 +1,1605 @@
...
+
+class WWVSIconSVGOptimizer extends enshrined\svgSanitize\Sanitizer {
```
Comment (grayside) [inline src/WWVSIconSVGOptimizer.class.inc:73]: Don't see this namespace declared anywhere.

**Review by grayside — CHANGES_REQUESTED** [review]
Overall this needs a bit more work on to become stable code, including more composer.json details, the (presumably) MIT license, and so on.

File: src/WWVSIconSVGOptimizer.class.inc (hunk 2 above)
Comment (ckoppenhaver) [reply src/WWVSIconSVGOptimizer.class.inc:71]: This is still a work in progress and the Drupal Specific code will be removed.

**Pull request closed without merging**